<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorsetter</class>
 <widget class="QDialog" name="colorsetter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>59</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 1</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="col1btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>59</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 2</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="col2btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>59</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 3</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="col3btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>59</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color 4</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="col4btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>colorsetter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>colorsetter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
